pennylane/devices/default_mixed.py

```diff
@@ -1057,4 +1072,47 @@ def preprocess(
         if self.readout_err is not None:
             transform_program.add_transform(warn_readout_error_state)
 
+        # Add the valicate section
+        transform_program.add_transform(validate_device_wires, self.wires, name=self.name)
+        transform_program.add_transform(validate_measurements, name=self.name)
+        transform_program.add_transform(
+            validate_observables, stopping_condition=observable_stopping_condition, name=self.name
+        )
+
         return transform_program, config
+
+
+# pylint: disable=too-many-positional-arguments, too-many-arguments
+def simulate(
+    circuit: qml.tape.QuantumScript,
+    rng=None,
+    prng_key=None,
+    debugger=None,
+    interface=None,
+    readout_errors=None,
+) -> Result:
+    """Simulate a single quantum script.
+
+    This is an internal function that will be called by ``default.qubit.mixed``.
+
+    Args:
+        circuit (QuantumTape): The single circuit to simulate
+        rng (Union[None, int, array_like[int], SeedSequence, BitGenerator, Generator]): A
+            seed-like parameter matching that of ``seed`` for ``numpy.random.default_rng``.
+            If no value is provided, a default RNG will be used.
+        prng_key (Optional[jax.random.PRNGKey]): An optional ``jax.random.PRNGKey``. This is
+            the key to the JAX pseudo random number generator. If None, a random key will be
+            generated. Only for simulation using JAX.
+        debugger (_Debugger): The debugger to use
+        interface (str): The machine learning interface to create the initial state with
+        readout_errors (List[Callable]): List of channels to apply to each wire being measured
+            to simulate readout errors.
+
+    Returns:
+        tuple(TensorLike): The results of the simulation
+
+    Note that this function can return measurements for non-commuting observables simultaneously.
+
+    This function assumes that all operations provide matrices.
+    """
+    raise NotImplementedError
```

Comment:
Is this going to be implemented in a different PR or something?

Reply:
In a different PR

Reply:
Here in the[ simulate logic of analytic case ](https://github.com/PennyLaneAI/pennylane/pull/6618)😆Please also join review if you are interested in it; I'm also looking for 2nd revewer for it but since it is in block by measure I haven't broadcast

Reply:
I'll mark this as resolved for now